sharepoint/3.5/modules/ROOT/pages/index.adoc

```diff
@@ -60,7 +60,16 @@ Configuring the connector to use OAuth v2.0 Authorization Code authentication re
 
 == OAuth Client Credentials 
 
-Configuring the connector to use OAuth Client Credentials with Certificate authentication requires the following information:
+To configure OAuth Client Credentials, follow these steps:
+
+. Create a keystore with a certificate (openssl).
+. Log in to https://portal.azure.com/#home[Microsoft Azure]. 
+. Go to *Azure Active Directory*.
+. Go to *App registrations*.
+. Add a new application and upload a certificate, or click on an existing application and click *Certificates & secrets > Certificates > Upload certificate* to upload a certificate.
+. Go to Anypoint Studio and create a new Mule project in which to add and configure the connector.
+. Add a dependency for SharePoint Connector in the `pom.xml` file.
+. Create the flow and add the keystore and the credentials file into `resources`. Add the required fields for the connector configuration. Configuring the connector to use OAuth Client Credentials with Certificate authentication requires the following information:
```

Comment:
Just a suggestion - it might be good to say what `resources` is - `resources` folder or `resources` directory